vulnerabilities/xxe/source/high.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+function HandleXmlError($errno, $errstr, $errfile, $errline){
+                    if ($errno==E_WARNING && (substr_count($errstr,"DOMDocument::loadXML()")>0))
+                    {
+                        throw new DOMException($errstr);
+                    }
+                    else
+                        return false;
+        }
+
+function XXE(){
+	$xmloutput = '';
+	if( isset( $_FILES[ 'uploaded' ] ) ) {
+		$filepath = $_FILES[ 'uploaded' ][ 'tmp_name' ];
+                $content = file_get_contents($filepath);
+                if(!preg_match_all('/http/i', $content)){
+		libxml_disable_entity_loader(false);
```

Comment:
Same issue here as with the medium one.

Reply:
I will change this exploit. 

I have written some simple code where a user could upload a .docx file. The xml parser will then parse the inner XML. Maybe thats a better one than the current.